Jenkinsfile

```diff
@@ -11,14 +11,36 @@ max_time = 120
 // assign any caught errors here
 err = null
 
+//Method to Kill PR Builds that are running currently when a new build is triggered.
+def abortPreviousRunningBuilds() {
+  def hi = Hudson.instance
+  def pname = env.JOB_NAME.split('/')[0]
```

Comment:
nit: better variable name, what does `pname` standfor? similary the variable `hi`